packages/app/src/cli/models/app/loader.ts

```diff
@@ -296,34 +297,32 @@ class AppLoader {
     })
     const configPaths = await path.glob(functionConfigPaths)
 
-    const functions = configPaths.map(async (configurationPath) => {
+    const allFunctions = configPaths.map(async (configurationPath) => {
       const directory = path.dirname(configurationPath)
-      const configuration = await this.parseConfigurationFile(FunctionExtensionConfigurationSchema, configurationPath)
+      const fileContent = await file.read(configurationPath)
+      const obj = toml.decode(fileContent)
+      const {type} = TypeSchema.parse(obj)
+      const spec = await functionSpecForType(type)
+      if (!spec) {
+        this.abortOrReport(
+          output.content`Unknown function type ${output.token.yellow(type)} in ${output.token.path(configurationPath)}`,
```

Comment:
Similar question here.  If we don't know the function type, in what situation would we continue?

Reply:
same as above 👌